tests/topotests/lib/topogen.py

```diff
@@ -676,6 +677,9 @@ def start(self):
 
         if result != '':
             self.tgen.set_error(result)
+        else:
+            for interface in self.links.keys():
```

Comment:
Maybe a comment here telling people you want to enable MPLS for all router interfaces?

Reply:
Indeed, I'll add a comment here.